src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -159,7 +159,7 @@ public function theStoreHasPaymentMethodNotAssignedToAnyChannel($paymentMethodNa
     public function thePaymentMethodRequiresAuthorizationBeforeCapturing(PaymentMethodInterface $paymentMethod)
     {
         $config = $paymentMethod->getGatewayConfig();
-        $config->setConfig(array_merge($config->getConfig(), ['use_authorize' => 1]));
+        $config->setConfig(array_merge($config->getConfig(), ['use_authorize' => true]));
```

Comment:
Backslash here too :)
\array_merge()

Reply:
Hello @Roshyo,

array_merge() is not in list of function which the PHP compiler optimizes.
You can see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php#L346 
and https://github.com/php/php-src/blob/f2db305fa4e9bd7d04d567822687ec714aedcdb5/Zend/zend_compile.c#L3872